models/audio/train_resnet.py

```diff
@@ -4,7 +4,8 @@
 os.environ["KMP_DUPLICATE_LIB_OK"] = "TRUE"
 
 from implementation.ResNet18 import ResNet18
-from train_network import train_val_split, plot_acc_loss
+from utils.train_network import train_val_split 
```

Comment:
na linuksie to tak nie dziala dlatego jest utils dodane. from draw_plot import plot_acc_loss

Reply:
and from train_network import train_val_split